apps/omg_eth/lib/eth/root_chain.ex

```diff
@@ -74,6 +75,17 @@ defmodule OMG.Eth.RootChain do
     )
   end
 
+  def piggyback_in_flight_exit(in_flightTx, outputIndex, from, contract \\ nil, opts \\ []) do
+    defaults =
+      @tx_defaults
+      |> Keyword.put(:gas, 1_000_000)
+      |> Keyword.put(:value, @piggyback_bond)
+
+    opts = defaults |> Keyword.merge(opts)
+    contract = contract || from_hex(Application.get_env(:omg_eth, :contract_addr))
+    Eth.contract_transact(from, contract, "piggybackInFlightExit(bytes, uint8)", [in_flightTx, outputIndex], opts)
```

Comment:
`piggybackInFlightExit(bytes, uint8)` is suspicious, normally the signatures don't contain spaces